### DISCUSSION TITLE
How to customize the navigation bar of my custom module?

### DISCUSSION THREAD

**Discussion by sarthakgaur** [directus/directus#7352] [Archive]
By custom module's navigation bar, I mean this:

![Nav](https://i.imgur.com/d73ebWP.png)

The navigation bar is currently empty, how can I customize it?

PS: The custom module is made using Vue components.

**Comment by rijkvanzanten** [comment]
@sarthakgaur There's a named `slot` you can use! 

In your Vue file, use:

```vue
<template>
  <private-view title="My Module">
    <template #navigation>
      This goes in the navigation bar!
    </template>
  </private-view>
</template>
```
